In the Prometheus chapter, advise to store  data locally (#1112)
uyuni-project/uyuni-docs: modules/administration/pages/monitoring.adoc

Otherwise you can download and install the package on your monitoring server manually.
The Prometheus software is also available for {productname} Proxy and {productname} {srm} Branch Server.

[NOTE]
----

> It is recommended to use a local filesystem for storing Prometheus data reliably.
> Non-POSIX compliant filesystems are not supported.
> NFS filesystems are not supported.

Reviewer: ```suggestion
Prometheus expects POSIX filesystem for storing data.
Non-POSIX compliant filesystems are not supported.
NFS filesystems are explicitly not supported.
```
